packages/asc-ui/src/components/Blog/BlogHeader/BlogHeader.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import BlogHeaderStyle from './BlogHeaderStyle'
-import Heading from '../../Typography/Heading'
+import Heading from '../../Heading/Heading'
 
 export type Props = {
   title: string
-}
+} & React.HTMLAttributes<HTMLElement>
```

Comment:
Replace HTMLAttributes type, whisch is huge with a small custom type/interface containing just the className